src/status_im2/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im2.contexts.wallet.send.select-address.tabs.view
+  (:require
+    [quo.core :as quo]
+    [react-native.gesture :as gesture]
+    [status-im2.contexts.wallet.send.select-address.tabs.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- render-account-item
+  [{:keys [color address] :as account}]
+  [quo/account-item
+   {:account-props (assoc account :customization-color color)
+    :on-press      (fn []
+                     (rf/dispatch [:wallet/select-send-account-address address])
+                     (rf/dispatch [:navigate-to-within-stack
+                                   [:wallet-select-asset :wallet-select-address]]))}])
+
+(def data
```

Comment:
I think this can have a better name than `data`. `tabs` or `tabs-data`. 

Also I think it is not used in that file?

Reply:
it's used in the main view, I can rename it 👌

Reply:
moved it to main view 👍